policies/gitlab/repository.rego

```diff
@@ -65,6 +65,7 @@ forking_allowed_for_repository = false {
 # custom:
 #   remediationSteps: [Make sure you have owner permissions, Go to the projects's settings -> Repository page, Enter "Protected branches" tab, select the default branch. Set the allowed to merge to "maintainers" and the allowed to push to "No one"]
 #   severity: MEDIUM
+#   prerequisites: [premium]
```

Comment:
Please add a general comment in the readme.md :
For free accounts, some policies will be skipped